core/embed/rust/src/ui/model_mercury/component/prompt_screen.rs

```diff
@@ -140,7 +101,12 @@ impl crate::ui::flow::Swipable for PromptScreen {}
 #[cfg(feature = "ui_debug")]
 impl crate::trace::Trace for PromptScreen {
     fn trace(&self, t: &mut dyn crate::trace::Tracer) {
-        t.component("PromptScreen");
-        t.child("button", &self.button);
+        t.component("StatusScreen");
```

Comment:
Why changing to `"StatusScreen"`?

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/c5abc8ad08b7f65bf22f7129a701b38c3c799b1c